examples/connectivity/v4/port2serviceprofilepublic/terraform.tf

```diff
@@ -1,8 +1,7 @@
 terraform {
   required_providers {
     equinix = {
-      source = "developer.equinix.com/terraform/equinix"
-      version = "9.0.0"
+      source = "equinix/equinix"
```

Comment:
I haven't checked to see if this is still necessary, but the reason this pointed to developer.equinix.com/terraform/equinix (at one point, perhaps still) is that [Terratest runs this test](https://github.com/equinix/terraform-provider-equinix/tree/master/tests) using a build of the Terraform plugin that is stashed in the home directory.  If `equinix/equinix` is not overridden in someway (there are other ways), the latest released plugin will be downloaded and the build of the local branch will not be used.

You've been in this area before with #277.
 
#279 may also be related.

Reply:
This change was meant to stay in #300 and has been rebased out of this PR.  This change brings the `port2serviceprofilepublic` config in line with all other examples; the test was failing in terratest because of the invalid provider specification.